lib/two_factor_authentication/models/two_factor_authenticatable.rb

```diff
@@ -30,14 +50,15 @@ def authenticate_otp(code, options = {})
           totp.verify_with_drift(code, drift)
         end
 
-        def otp_code(time = Time.now, options = {})
+        def generate_totp_code(time = Time.now, options = {})
```

Comment:
I don't see this method being used anywhere. Unless I missed it, I couldn't find a test for the Google Authenticator scenario. I would assume that a controller or feature spec would call this method to allow the user to enter the code from their Authenticator app.

Also, I would call this method `totp_code`, not `generate_totp_code` because what this method returns is the user's TOTP code at the current time. It does not generate a new code.